src/H5ACmpio.c

```diff
@@ -2104,31 +2099,28 @@ H5AC__run_sync_point(H5F_t *f, int sync_point_op)
 
     /* Sanity checks */
     HDassert(f != NULL);
-
     cache_ptr = f->shared->cache;
-
     HDassert(cache_ptr != NULL);
-
     aux_ptr = (H5AC_aux_t *)H5C_get_aux_ptr(cache_ptr);
-
     HDassert(aux_ptr != NULL);
     HDassert(aux_ptr->magic == H5AC__H5AC_AUX_T_MAGIC);
     HDassert((sync_point_op == H5AC_SYNC_POINT_OP__FLUSH_TO_MIN_CLEAN) ||
              (sync_point_op == H5AC_METADATA_WRITE_STRATEGY__DISTRIBUTED));
 
 #ifdef H5AC_DEBUG_DIRTY_BYTES_CREATION
-    HDfprintf(stdout, "%d:H5AC_propagate...:%u: (u/uu/i/iu/m/mu) = %zu/%u/%zu/%u/%zu/%u\n", aux_ptr->mpi_rank,
-              aux_ptr->dirty_bytes_propagations, aux_ptr->unprotect_dirty_bytes,
+    HDfprintf(stdout, "%d:%s...:%u: (u/uu/i/iu/m/mu) = %zu/%u/%zu/%u/%zu/%u\n", aux_ptr->mpi_rank,
+              __func__ aux_ptr->dirty_bytes_propagations, aux_ptr->unprotect_dirty_bytes,
```

Comment:
Looks like a comma was missed here in the develop PR..

Reply:
Fixed. Will fix in develop.